libs/shared/src/components/CopyToClipboard/index.tsx

```diff
@@ -28,6 +31,8 @@ export const CopyToClipboard: FunctionComponent<Props> = (props) => {
 
   const onClick = (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
     Clipboard.copy(props.payload);
+    setOpen(true);
+    setTimeout(() => setOpen(false), 7000);
     event.stopPropagation();
```

Comment:
stories are awesome for catching little UX issues like this one (notificiation did not show if clicking on text tobe copied)